### DISCUSSION TITLE
Sveltekit example without typescript

### DISCUSSION THREAD

**Discussion by YerayAlonso** [vercel/community#801] [Idea]
I'm finding issues when trying to disable typescript from the sveltekit example.

Wouldn't be interesting to have an example without typescript to start coding without having to play with packages and config files?

**Comment by amyegan** [comment]
Hi @YerayAlonso. Can you share a link to the SvelteKit example you're talking about? I would like to get a better idea of where I should look and what we could do to make things easier.

**Reply by YerayAlonso** [reply]
Hi @amyegan.
Sorry, I mean the example project [here](https://github.com/vercel/vercel/tree/main/examples/sveltekit), which seems to be the one that is used if you select SvelteKit template when creating a new project in vercel's web [here](https://vercel.com/new/templates).

**Reply by amyegan** [reply]
Thanks for sharing these links! I thought that was the most likely example, but didn't want to assume the wrong thing. 

Can you tell me more about the issues you're facing? I am able to use regular JS code in the starter template without making any config changes, but I might not be looking in the right place to replicate what you're seeing.

**Reply by YerayAlonso** [reply]
Sorry for the late reply.

If I create the project as [sveltekit docs](https://kit.svelte.dev/docs/introduction) suggests (`npm create svelte@latest my-app`), and choose to use no typescript, I get a project without [app.d.ts](https://github.com/vercel/vercel/blob/main/examples/sveltekit/src/app.d.ts) and that made me think I was forced to use typescript when using vercel's template.

> I am able to use regular JS code in the starter template without making any config changes

That's fine for me then.  
Thanks.
